### PR TITLE
Fyle-cxczcr Transactions list in Personal Cards

### PR DESCRIPTION
**Pull request by iravisankar**
localhost_8100_enterprise_personal_cards(iPhone X) (6)
localhost_8100_enterprise_personal_cards(iPhone X) (7)

### CODE REVIEW

File: src/app/fyle/personal-cards/personal-cards.page.html [hunk 1]
```diff
@@ -36,8 +41,52 @@
         *ngIf="!isLoading"
         [linkedAccounts]="linkedAccounts"
         (deleted)="onDeleted()"
+        (changed)="onChanged($event)"
       ></app-bank-account-cards>
     </ng-container>
+
+    <ng-container *ngIf="isCardsLoaded">
```
Comment (Dimple16) [inline src/app/fyle/personal-cards/personal-cards.page.html:48]: `isCardsLoaded` - can we change this naming to `isLoading` / `isCardsLoading`?

File: src/app/fyle/personal-cards/personal-cards.page.html [hunk 2]
```diff
@@ -36,8 +41,52 @@
...
+              [txnDate]="transaction.btxn_created_at"
+              [previousTxnDate]="transactions[i-1]?.btxn_created_at"
+            >
+            </app-personal-card-transaction>
+          </ng-container>
+        </ng-container>
+      </ng-container>
+    </ng-container>
+  </ng-container>
+  <ng-container *ngIf="((linkedAccountsCount$ | async) !== 0) && isTrasactionsLoading || isLoadingDataInInfiniteScroll">
```
Comment (Dimple16) [inline src/app/fyle/personal-cards/personal-cards.page.html:80]: `(linkedAccountsCount$ | async) > 0` works?

Comment (Dimple16) [reply src/app/fyle/personal-cards/personal-cards.page.html:80]: `isLoadingDataInInfiniteScroll` typo

File: src/global.scss [hunk 3]
```diff
@@ -954,16 +965,16 @@ ion-modal.auto-height {
 .fyle.swiper-pagination-bullet {
   width: 6px !important;
   height: 6px !important;
-  background: #fff !important;
-  border: 1px solid #161528 !important;
+  background: $pure-white !important;
```
[the remark this answers is not in the document]
Comment (iravisankar) [reply src/global.scss:968]: yes .., if not added, it's not working...

**Review by pradeep8093 — APPROVED** [review]

File: src/app/core/services/personal-cards.service.ts [hunk 4]
```diff
@@ -59,4 +59,42 @@ export class PersonalCardsService {
   deleteAccount(accountId: string): Observable<PersonalCard> {
     return this.expenseAggregationService.delete('/bank_accounts/' + accountId);
   }
+
+  getBankTransactions(
+    config: Partial<{ offset: number; limit: number; order: string; queryParams: any }> = {
```
Comment (Julias0) [inline src/app/core/services/personal-cards.service.ts:64]: queryParams: {accountId: string, status: string}

File: src/app/core/services/personal-cards.service.ts [hunk 5]
```diff
@@ -59,4 +59,42 @@ export class PersonalCardsService {
...
+        btxn_status: config.queryParams.status,
+        limit: config.limit,
+        offset: config.offset,
+      },
+    });
+  }
+
+  getBankTransactionsCount(queryParams) {
+    const params = {
+      limit: 10,
```
Comment (Julias0) [inline src/app/core/services/personal-cards.service.ts:82]: limit should be `1` for count call

**Review by Julias0 — APPROVED** [review]

**Pull request merged**